### PR TITLE
Arhetype

### LINKED ISSUE #83: Arkctl 增加模块代码初始化能力
用户使用时，新建模块时，需要全新生成模块代码，可以通过 Arkctl 完成模块代码仓库的新建。

### Feature description
1. 需要知道一个模块的代码是怎样的
2. arkctl 通过执行 `mvn archetype generate` 生成模块代码

### PR DESCRIPTION
**Pull request by sususama**
### Motivation

Explain the context, and why you're making that change.
To make others understand what is the problem you're trying to solve.

### Modification

Describe the idea and modifications you've done.

### Result

Resolved or fixed #83 

If there is no issue then describe the changes introduced by this PR.

### CODE REVIEW

File: sofa-serverless-runtime/pom.xml [hunk 1]
```diff
@@ -82,6 +82,7 @@
         <module>arklet-core</module>
         <module>arklet-springboot-starter</module>
         <module>sofa-serverless-adapter-ext</module>
+        <module>sofa-maven-archetype</module>
     </modules>
```
Comment (yuanyuancin) [inline sofa-serverless-runtime/pom.xml:86]: 可以叫sofa-serverless-biz-maven-archetype

**Comment by lvjing2** [comment]
需要补充下使用文档

**Comment by sususama** [comment]
> 需要补充下使用文档

具体补充在哪里？

File: sofa-serverless-runtime/sofa-serverless-biz-maven-archetype/pom.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                    <goals>
+                        <goal>repackage</goal>
+                    </goals>
+                </execution>
+            </executions>
+            <configuration>
+                <skipArkExecutable>true</skipArkExecutable>
+                <outputDirectory>./target</outputDirectory>
+                <bizName>biz1</bizName>
+                <webContextPath>biz1</webContextPath>
```
Comment (yuanyuancin) [inline sofa-serverless-runtime/sofa-serverless-biz-maven-archetype/pom.xml:50]: bizName、webContextPath可以用用户传入的 app参数

**Pull request closed without merging**